src/metabase/pulse/render/body.clj

```diff
@@ -548,31 +539,6 @@
                                 :width   :100%})
            :src   (:image-src image-bundle)}]]})
 
-(defn- render-multiple-lab-chart
```

Comment:
This function is moved down below since it uses `single-x-axis-combo-series` and `double-x-axis-combo-series` which are defined below this line.